docs/metrics/prometheus/README.md

```diff
@@ -104,32 +104,76 @@ Here `prometheus.yml` is the name of configuration file. You can now see MinIO m
-- `minio_total_disks` : Total number of disks for current MinIO server instance
-- `minio_disk_storage_available_bytes` : Current storage space available to MinIO server in bytes
-- `minio_disk_storage_total_bytes` : Total storage space available to MinIO server in bytes
-- `process_start_time_seconds` : Start time of MinIO server since unix epoc hin seconds
-
-If you're running MinIO gateway, disk/storage information is not exposed. Only following metrics are available
-
-- `minio_http_requests_duration_seconds_bucket` : Cumulative counters for all the request types (HEAD/GET/PUT/POST/DELETE) in different time brackets
-- `minio_http_requests_duration_seconds_count` : Count of current number of observations i.e. total HTTP requests (HEAD/GET/PUT/POST/DELETE)
-- `minio_http_requests_duration_seconds_sum` : Current aggregate time spent servicing all HTTP requests (HEAD/GET/PUT/POST/DELETE) in seconds
-- `minio_network_received_bytes_total` : Total number of bytes received by current MinIO server instance
-- `minio_network_sent_bytes_total` : Total number of bytes sent by current MinIO server instance
-- `process_start_time_seconds` : Start time of MinIO server since unix epoch in seconds
-
-For MinIO instances with [`caching`](https://github.com/minio/minio/tree/master/docs/disk-caching) enabled, these additional metrics are available.
-
-- `minio_disk_cache_storage_bytes` : Total byte count of cache capacity available for current MinIO server instance
-- `minio_disk_cache_storage_free_bytes` : Total byte count of free cache available for current MinIO server instance
+- `minio_disk_storage_used` : Disk space used by the disk.
+- `minio_disk_storage_available`: Available disk space left on the disk.
+- `minio_disk_storage_total`: Total disk space on the disk.
+- `minio_disks_offline`: Total number of offline disks in current MinIO instance.
+- `minio_disks_total`: Total number of disks in current MinIO instance.
+- `minio_http_s3_requests_total`: Total number of s3 requests in current MinIO instance.
+- `minio_http_s3_errors_total`: Total number of errors in s3 requests in current MinIO instance.
+- `minio_http_s3_requests_current`: Total number of active s3 requests in current MinIO instance.
+- `minio_http_internode_rx_bytes_total`: Total number of internode bytes received by current MinIO server instance.
+- `minio_http_internode_tx_bytes_total`: Total number of bytes sent to the other nodes by current MinIO server instance.
+- `minio_http_s3_rx_bytes_total`: Total number of s3 bytes received by current MinIO server instance.
+- `minio_http_s3_tx_bytes_total`: Total number of s3 bytes sent by current MinIO server instance.
+- `minio_version_info`: Current MinIO version with commit-id.
+- `minio_http_requests_duration_seconds`: Histogram that holds the latency information of the requests.
+
+
+## Migration guide for the new set of metrics
+
+This migration guide applies for the version upgrade from <RELEASE_VERSION> TO <RELEASE_VERSION>
```

Comment:
Don't need to mention version, just say migration guide for older releases any releases before today i.e this Wed.